### PR TITLE
Add CVE-2021-4176 for 8b531ae9-2d36-43ff-af33-4d81acfb2f27

### PR DESCRIPTION
**Pull request by JamieSlome**
Add CVE-2021-4176 for [8b531ae9-2d36-43ff-af33-4d81acfb2f27](https://huntr.dev/bounties/8b531ae9-2d36-43ff-af33-4d81acfb2f27) @huntr-helper

### CODE REVIEW

File: 2021/4xxx/CVE-2021-4176.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
```
Comment (cve-team) [inline 2021/4xxx/CVE-2021-4176.json:1]: None is not of type 'string'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
@JamieSlome Please resolution the validation issues and the merge conflict. Once resolved we can process and merge request.

**Comment by JamieSlome** [comment]
@anthnysingleton - sorted! ❤️

**Pull request merged**